packages/cli/generators/app/index.js

```diff
@@ -59,7 +59,7 @@ module.exports = class AppGenerator extends ProjectGenerator {
 
   async setOptions() {
     await super.setOptions();
-    if (this.shouldExit()) return;
+    if (this.shouldExit()) return false;
```

Comment:
This change is not needed. Let's revert to make the diff cleaner.

Reply:
Done